backend/src/Wasm/Libs/HttpClient.fs

```diff
@@ -88,6 +89,14 @@ module HttpClient =
               VersionPolicy = System.Net.Http.HttpVersionPolicy.RequestVersionOrLower
             )
 
+          let strBody = System.Text.Encoding.UTF8.GetString httpRequest.body
+
+          // CLEANUP: BodyContent, used by the 'regular' HttpClient,
+          // doesn't seem to work in the Wasm runtime,
+          // so we'll have to figure out some other way of supporting
+          // non-text bodies in that environment
+          if strBody <> "" then req.Content <- new StringContent(strBody)
```

Comment:
I took a look at the Fetch API and it seems to have support for bytes, so this isn't a fundamental problem.